test/extended/images/mysql_replica.go

```diff
@@ -0,0 +1,266 @@
+package images
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/util/wait"
+)
+
+func selector(selector string) labels.Selector {
```

Comment:
why this is a func? it looks pretty generic to be special function... just call labels.Parse()